controllers/userControllers.js

```diff
@@ -57,6 +57,25 @@ exports.readUser = async (req, res) => {
   }
 };
 
-exports.updateUser = async (req, res) => {};
+exports.updateUser = async (req, res) => {
+  const { id, email, password, permission } = req.body;
+  try {
+    const [updatedUser] = await database.User.update(
+      {
+        ...(email && { email }),
+        ...(password && { password }),
+        ...(permission && { permission }),
+      },
+      { where: { id } }
+    );
+    if (updatedUser === 0) {
+      return res.status(404).send("User not found");
```

Comment:
I'm curious on why are you using `.send` while using `.json` on all other methods (delete, read, etc)?